packages/state-manager/src/sagas/messages/messages.selectors.ts

```diff
@@ -45,6 +45,22 @@ export const publicChannelMessagesEntities = (address: string) =>
     return channelMessagesAdapter.getSelectors().selectEntities(channelMessagesBase.messages)
   })
 
+export const getMessagesFromChannelIdByPubKey = (channelId: string, pubkey: string) =>
```

Comment:
I don't think it's actually necessary to have such selector. See my comments in `verifyMessages.saga`